nixos/modules/misc/ids.nix

```diff
@@ -347,6 +347,7 @@ in
       #mailman = 316;  # removed 2019-08-30
       zigbee2mqtt = 317;
       # shadow = 318; # unused
+      moonraker = 319;
```

Comment:
Why is DynamicUser not enough?

Reply:
Moonraker keeps its own database and also requires read-write privileges to a persistent directory.